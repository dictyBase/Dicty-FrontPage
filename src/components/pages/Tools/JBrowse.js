// @flow
import React from "react"
import { Flex, Box } from "rebass"
import { Link } from "styles"

/**
 * This displays genome browser links.
 */

const JBrowse = () => {
  return (
    <Flex justify="center">
      <Box style={{ textAlign: "center" }}>
        <h1>
          <strong>Genome Browser</strong>
        </h1>
        <p>
          {" "}
          The following genomes are currently available on the new genome
          browser:
          <br />
          <br />{" "}
        </p>
        <em>
          <p>
            <Link
              href="http://dictybase.org/tools/jbrowse/?data=data%2Fjbrowse%2Fdiscoideum&amp;loc=1%3A2334653..2521452&amp;tracks=transcript%2CtRNA%2Cgene%2Cretino2&amp;highlight="
              title="D. discoideum"
              target="new">
              D. discoideum
            </Link>
          </p>
          <p>
            <Link
              href="http://dictybase.org/tools/jbrowse/?data=data%2Fjbrowse%2Fpurpureum&amp;loc=scaffold_1%3A89486..146533&amp;tracks=&amp;highlight="
              title="D. purpureum"
              target="new">
              D. purpureum
            </Link>
          </p>
          <p>
            <Link
              href="http://dictybase.org/tools/jbrowse/?data=data%2Fjbrowse%2Fpallidum&amp;loc=PPA1267848%3A582884..992552&amp;tracks=tblastn_discoideum%2Ctranscript%2Cgene&amp;highlight="
              title="P. pallidum"
              target="new">
              P. pallidum
            </Link>
          </p>
          <p>
            <Link
              href="http://dictybase.org/tools/jbrowse/?data=data%2Fjbrowse%2Ffasciculatum&amp;loc=DFA1435654%3A879778..900543&amp;tracks=gene%2CtRNA%2Ctranscript%2Ctblastn_discoideum%2Creference&amp;highlight="
              title="D. fasciculatum"
              target="new">
              D. fasciculatum
            </Link>
          </p>
        </em>
      </Box>
    </Flex>
  )
}

export default JBrowse
