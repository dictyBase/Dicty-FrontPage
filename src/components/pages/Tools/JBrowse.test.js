import React from "react"
import toJson from "enzyme-to-json"
import { shallow } from "enzyme"
import "jest-styled-components"
import JBrowse from "./JBrowse"

test("matching a snapshot of JBrowse", () => {
  const subject = shallow(<JBrowse />)
  expect(toJson(subject)).toMatchSnapshot()
})
