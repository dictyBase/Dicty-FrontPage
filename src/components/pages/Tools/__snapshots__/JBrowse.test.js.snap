// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matching a snapshot of JBrowse 1`] = `
<Flex
  justify="center"
>
  <Box
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <h1>
      <strong>
        Genome Browser
      </strong>
    </h1>
    <p>
       
      The following genomes are currently available on the new genome browser:
      <br />
      <br />
       
    </p>
    <em>
      <p>
        <Link
          href="http://dictybase.org/tools/jbrowse/?data=data%2Fjbrowse%2Fdiscoideum&loc=1%3A2334627..2521426&tracks=transcript%2CtRNA%2Cgene%2Cretino2&highlight="
          target="new"
          title="D. discoideum"
        >
          D. discoideum
        </Link>
      </p>
      <p>
        <Link
          href="http://dictybase.org/tools/jbrowse/?data=data%2Fjbrowse%2Fpurpureum&loc=scaffold_1%3A89495..146542&tracks=&highlight="
          target="new"
          title="D. purpureum"
        >
          D. purpureum
        </Link>
      </p>
      <p>
        <Link
          href="http://dictybase.org/tools/jbrowse/?data=data%2Fjbrowse%2Fpallidum&loc=PPA1267848%3A582840..992508&tracks=tblastn_discoideum%2Ctranscript%2Cgene&highlight="
          target="new"
          title="P. pallidum"
        >
          P. pallidum
        </Link>
      </p>
      <p>
        <Link
          href="http://dictybase.org/tools/jbrowse/?data=data%2Fjbrowse%2Ffasciculatum&loc=DFA1435654%3A879771..900536&tracks=gene%2CtRNA%2Ctranscript%2Ctblastn_discoideum%2Creference&highlight="
          target="new"
          title="D. fasciculatum"
        >
          D. fasciculatum
        </Link>
      </p>
    </em>
  </Box>
</Flex>
`;
